perf/perftool.py

```diff
@@ -32,16 +30,24 @@ class Perftool(Test):
 
     def setUp(self):
         '''
-        Build perftool Test
-        Source:
-        https://github.com/rfmvh/perftool-testsuite
+        Install the basic packages to support perf
         '''
 
         # Check for basic utilities
         smm = SoftwareManager()
-        for package in ['gcc', 'make', 'perf']:
+        detected_distro = distro.detect()
+        kernel_ver = platform.uname()[2]
+        deps = ['gcc', 'make']
+        if 'Ubuntu' in detected_distro.name:
+            deps.extend(['linux-tools-common', 'linux-tools-%s' % kernel_ver])
```

Comment:
For ubuntu, perf is installed using linux-tools-common, perf package is not needed.